compiler/src/org.graalvm.compiler.replacements.test/src/org/graalvm/compiler/replacements/test/StringCompareToTest.java

```diff
@@ -48,10 +48,12 @@
                     "crazy dog jumps over laszy fox"
     };
 
-    /**
-     * Initialize variables
-     */
     public StringCompareToTest() {
+        if (Java8OrEarlier) {
```

Comment:
Instead of this, use `Assume.assumeFalse(Java8OrEarlier);` (e.g., https://github.com/oracle/graal/blob/bfe765f1b04d9dc0dc25cac6f4073115b21bf533/compiler/src/org.graalvm.compiler.debug.test/src/org/graalvm/compiler/debug/test/DebugContextTest.java#L229).